lib/hawkular/inventory/inventory_api.rb

```diff
@@ -25,9 +26,9 @@ def initialize(entrypoint = nil, credentials = {}, options = {})
     #   entrypoint: http://localhost:8080/hawkular/inventory
     # and another sub-hash containing the hash with username[String], password[String], token(optional)
     def self.create(hash)
-      hash[:entrypoint] ||= 'http://localhost:8080/hawkular/inventory'
       hash[:credentials] ||= {}
```

Comment:
perhaps worth to unify the code which does the entrypoint/credentials assignment among subclients? it feels like copy paste... @pilhuhn